src/main/java/com/redhat/developers/msa/ola/OlaController.java

```diff
@@ -104,22 +104,35 @@ public String health() {
      * @return The feign pointing to the service URL and with Hystrix fallback.
      */
     private HolaService getNextService() {
-        // This stores the Original/Parent ServerSpan from ZiPkin.
-        final ServerSpan serverSpan = brave.serverSpanThreadBinder().getCurrentServerSpan();
-        final CloseableHttpClient httpclient =
-            HttpClients.custom()
-                .addInterceptorFirst(new BraveHttpRequestInterceptor(brave.clientRequestInterceptor(), new DefaultSpanNameProvider()))
-                .addInterceptorFirst(new BraveHttpResponseInterceptor(brave.clientResponseInterceptor()))
-                .build();
+        String url = System.getenv("HOLA_SERVER_URL");
```

Comment:
Ok sounds reasonable - possibly would be worth moving to utility class just to simplify this code.